deploy/kubernetes/README.md

```diff
@@ -1,18 +1,37 @@
-1. One-time per project: Create GCP service account for CSI driver
-    1. Export environment variables for location for service account private key file and name of the service account
+1. [One-time per project] Create GCP service account credential for CSI Driver and download JSON:
+    1. Go to https://pantheon.corp.google.com/apis/credentials
+    2. Select `Create credentials` -> ~Service account key`
+    3. For `Service Account` select `Compute Engine Default service account`
+    4. Select `JSON`
+    5. Click `Create` to download JSON for the private key to your local machine.
```

Comment:
You shouldn't have to download keys from pantheon. The script should already create the service account and key.

Reply:
@davidz627 could you clarify? Are these steps to create and download the key required? If not, is the target JSON path something that will be created?

Reply:
there is a script that does substeps 1-6, it is called `setup_project.sh`. As long as you set all the right env variables the script will handle creating service account, binding roles, and downloading the json key to the provided directory (`SA_FILE`)